madetech/serveless-pipelines-poc
# Added MVP end-to-end testing for each pipeline step

Test script will, for each pipeline in turn:
* Download latest input files from GCP bucket
* Download latest output file from GCP bucket
* Execute `run_sql_save_res_to_parquet`
* Diff resulting output from latest output pulled from bucket and assert equality

Only changes to existing code was to make some imports explicitly relative

Added a `requirements.txt` that covers pipeline dependencies and test dependencies. Note that this approach won't work if the dependencies of the individual pipelines diverge
